genie-web/src/main/java/com/netflix/genie/web/scripts/CommandSelectorManagedScript.java

```diff
@@ -67,33 +64,18 @@ public CommandSelectorManagedScript(
     ) throws ResourceSelectionException {
         log.debug("Called to attempt to select a command from {} for job {}", resources, jobRequest);
 
-        try {
-            final Object evaluationResult = this.evaluateScript(
-                ImmutableMap.of(
-                    JOB_REQUEST_BINDING, jobRequest,
-                    COMMANDS_BINDING, resources
-                )
-            );
-            if (!(evaluationResult instanceof ResourceSelectorScriptResult)) {
-                throw new ResourceSelectionException(
-                    "Command selector evaluation returned invalid type: " + evaluationResult.getClass().getName()
-                );
-            }
-            @SuppressWarnings("unchecked") final ResourceSelectorScriptResult<Command> result
-                = (ResourceSelectorScriptResult<Command>) evaluationResult;
-
-            // Validate that the selected resource is actually in the original set
-            if (result.getResource().isPresent() && !resources.contains(result.getResource().get())) {
-                throw new ResourceSelectionException(result.getResource().get() + " is not in original set");
-            }
+        return super.selectResource(resources, jobRequest);
+    }
 
-            return result;
-        } catch (
-            final ScriptExecutionException
-                | ScriptNotConfiguredException
-                | RuntimeException e
-        ) {
-            throw new ResourceSelectionException(e);
-        }
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    protected void addParametersForScript(
+        final Map<String, Object> parameters,
+        final Set<Command> resources,
+        final JobRequest jobRequest
+    ) {
+        parameters.put(COMMANDS_BINDING, resources);
```

Comment:
mostly cause they're bound (currently) under different names. I thought about making them just be `resources` and putting them in there blindly but then I thought it might be confusing for script developers so I kind of left it